.github/workflows/release.yml

```diff
@@ -0,0 +1,68 @@
+name: Release Charts
+
+on: 
+  workflow_dispatch:
+    inputs:
+      csi_operator_version:
+        description: 'Last released CSI Operator version'
+        required: true
+      release_tag:
+        description: 'Current release tag'
+        required: true
+
+jobs:
+  build:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+      
+      - name: Make version
+        id: vars
+        run: |
+          echo "::set-output name=version::$(make version)"
+
+      - name: Create asset
+        run: |
+          PACKAGE_NAME=csi-baremetal-deployment-${{ github.event.inputs.csi_operator_version }}.tgz
+          DIR=./csi-baremetal-deployment-${{ steps.vars.outputs.version }}
+          cp -r ./charts $DIR
+          tar -czvf ./$PACKAGE_NAME $DIR
```

Comment:
To make package you should use helm command

Reply:
Ok, and what about version? Do I need to use only operator version for `helm package --version=${{ github.event.inputs.csi_operator_version }}`?